.github/workflows/nodejs.yml

```diff
@@ -1,15 +1,15 @@
 name: Node.js CI
 
-on: [push, pull_request]
+on: [push, pull_request, workflow_dispatch]
 
 jobs:
   build:
     runs-on: ubuntu-latest
 
     steps:
-      - uses: actions/checkout@v3
-      - uses: actions/setup-node@v3
+      - uses: actions/checkout@v4
+      - uses: actions/setup-node@v4
         with:
-          node-version: 20
+          node-version: 22
```

Comment:
Could you revert this, or test on both v20 and v22?

Reply:
Sure, will test on 20